src/content/docs/dropins/user-auth/recaptcha.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: reCAPTCHA Integration
```

Comment:
I think that sentence case is the convention across repos for topic titles since it's typically the format we use for headings in the topic body.